modules/ROOT/pages/_partials/symbolicate-net.adoc

```diff
@@ -0,0 +1,17 @@
+// BEGIN -- inclusion -- {root-partials}symbolicate-net.adoc
```

Comment:
I don't think this section is applicable for .NET. .NET will have their own way for symbolicating the symbols. Please check with @borrrden .

Reply:
Turns out that this file didn't have a menu item, so I think it was deleted on purpose. I've removed the file.